File: app/src/main/java/ir/touristland/Models/HotelPrices/TotalPrice.java
package ir.touristland.Models.HotelPrices;

import com.google.gson.annotations.SerializedName;

public class TotalPrice {

    @SerializedName("webservice")
    private int webservice;

    @SerializedName("sales")
    private int sales;

    public int getWebservice() {
        return webservice;
    }

    public void setWebservice(int webservice) {
        this.webservice = webservice;
    }

    public int getSales() {
        return sales;
    }

    public void setSales(int sales) {
        this.sales = sales;
    }

    @Override
    public String toString() {
        return
                "TotalPrice{" +
                        "webservice = '" + webservice + '\'' +
                        ",sales = '" + sales + '\'' +
                        "}";
    }
}

File: app/src/main/java/ir/touristland/Models/HotelPrices/TotalPrice.kt
package ir.touristland.Models.HotelPrices

import com.google.gson.annotations.SerializedName

class TotalPrice {

    @SerializedName("webservice")
    var webservice: Int = 0

    @SerializedName("sales")
    var sales: Int = 0

    override fun toString(): String {
        return "TotalPrice{" +
                "webservice = '" + webservice + '\''.toString() +
                ",sales = '" + sales + '\''.toString() +
                "}"
    }
}